leaderboard.py

```diff
@@ -20,7 +21,7 @@
   firebase_admin.initialize_app(credentials.Certificate(get_credentials_json()))
   db = firestore.client()
 
-SUPPORTED_TRANSLATION_LANGUAGES = [
+SUPPORTED_LANGUAGES = [
```

Comment:
Is it going to deal with translation only? so it's clear the supported languages will be only about translation?

wondering why you removed `translation`.

Reply:
no it's used in summarization too. so it's for the entire app.

Reply:
will they always support the same languages? if so I think it'll be fine.

Reply:
yep